### ISSUE TITLE
Accordion examples - scrollable region must have keyboard access

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#5124]
The [Fixed example](https://www.patternfly.org/v4/components/accordion/html#fixed) has an expanded accordion with a scrollbar that cannot receive keyboard focus.

**Comment by RuiBrian** [comment]
I can be assigned to this issue!
